.gitignore

```diff
@@ -110,3 +110,5 @@ share/python-wheels/
 .installed.cfg
 *.egg
 # MANIFEST
+MANIFEST
```

Comment:
I intentionally disabled `MANIFEST` to avoid git warnings about non-Python files. Let me check how to improve this line to apply only to Python.